CHANGES.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+### _Black_
+
+- Reformat zero-length and all-whitespace files as a single empty line, i.e. just a
+  linefeed character (#2382)
```

Comment:
```suggestion
  linefeed character (#2484)
```
This should be the PR number, not the issue number.

Also, thanks for submitting this PR! I'll take a deeper look soon.

Reply:
Hi @JelleZijlstra,

I amended and force-pushed this correction as soon as the linter warned me about it – I didn't expect any human to notice it this soon :)

I hope the force-push didn't mess anything up.